test/linalg/bidiag.jl

```diff
@@ -40,6 +45,32 @@ for relty in (Float32, Float64, BigFloat), elty in (relty, Complex{relty})
         x = T \ b
         tx = Tfull \ b
         @test norm(x-tx,Inf) <= 4*condT*max(eps()*norm(tx,Inf), eps(relty)*norm(x,Inf))
+        @test_throws DimensionMismatch T \ ones(elty,n+1,2)
+        @test_throws DimensionMismatch T.' \ ones(elty,n+1,2)
+        @test_throws DimensionMismatch T' \ ones(elty,n+1,2)
+        if relty != BigFloat
+            x = T.'\c.'
+            tx = Tfull.' \ c.'
+            @test norm(x-tx,Inf) <= 4*condT*max(eps()*norm(tx,Inf), eps(relty)*norm(x,Inf))
+            @test_throws DimensionMismatch T.'\b.'
+            x = T'\c.'
+            tx = Tfull' \ c.'
+            @test norm(x-tx,Inf) <= 4*condT*max(eps()*norm(tx,Inf), eps(relty)*norm(x,Inf))
+            @test_throws DimensionMismatch T'\b.'
+            x = T\c.'
+            tx = Tfull\c.'
+            @test norm(x-tx,Inf) <= 4*condT*max(eps()*norm(tx,Inf), eps(relty)*norm(x,Inf))
+            @test_throws DimensionMismatch T\b.'
+        end
+
+        debug && println("Generic Mat-vec ops")
+        @test_approx_eq T*dv Tfull*dv
+        @test_approx_eq T'*dv Tfull'*dv
+        @test_approx_eq T/dv' Tfull/dv'
```

Comment:
@kshyatt Do you remember why you added this test? Is it on purpose that you test `/` and not `\`?